core/storage/user/gae_models.py

```diff
@@ -321,3 +321,43 @@ def get_or_create(cls, user_id, collection_id):
             return instance_model
         else:
             return cls.create(user_id, collection_id)
+
+
+class UserQueryModel(base_models.BaseModel):
+    """Model for storing result of queries.
+
+    The id of each instance of this model is alphanumeric id of length 12
+    unique to each model instance.
+    """
+    # Options for a query specified by query submitter.
+    # Query option to specify whether user has created or edited one or more
+    # explorations in last n days.
```

Comment:
Add: This only returns users who have ever created or edited at least one exploration (which I believe matches your impl).